atomicapp/providers/openshift.py

```diff
@@ -313,6 +365,9 @@ def _parse_kubeconf(self, filename):
             if co["name"] == current_context:
                 context = co
 
+        if not context:
+            raise ProviderFailedException()
```

Comment:
Don't really understand why this change would made. Please change it back if there was not good reason.